npm run bootstrap err

Hi,I get this error when I use npm run bootstrap.Please tell me where something went wrong.Thank you！

> > caliper@0.1.0 bootstrap /home/zhuliang/performance-caliper/caliper
> lerna bootstrap

lerna info version 2.0.0
lerna info Bootstrapping 8 packages
lerna info lifecycle preinstall
lerna info Installing external dependencies
lerna info hoist Installing hoisted dependencies into root
lerna info hoist Pruning hoisted dependencies
lerna info hoist Finished pruning hoisted dependencies
lerna ERR! execute Error: Command failed: npm install
lerna ERR! execute npm WARN deprecated grpc-web-client@0.7.0: This package has been deprecated in favor of @improbable-eng/grpc-web
lerna ERR! execute npm WARN deprecated hoek@4.2.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
lerna ERR! execute npm WARN deprecated formatio@1.2.0: This package is unmaintained. Use @sinonjs/formatio instead
lerna ERR! execute npm WARN deprecated samsam@1.3.0: This package has been deprecated in favour of @sinonjs/samsam
lerna ERR! execute npm WARN deprecated text-encoding@0.6.4: no longer maintained
lerna ERR! execute npm WARN deprecated circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.
lerna ERR! execute npm WARN deprecated core-js@2.3.0: core-js@<2.6.6 is no longer maintained. Please, upgrade to core-js@3 or at least to actual version of core-js@2.
lerna ERR! execute npm WARN deprecated hawk@6.0.2: This version has been deprecated. Please upgrade to the latest version to get the best features, bug fixes, and security patches.
lerna ERR! execute npm WARN deprecated boom@4.3.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
lerna ERR! execute npm WARN deprecated cryptiles@3.1.4: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
lerna ERR! execute npm WARN deprecated boom@5.2.0: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
lerna ERR! execute npm WARN deprecated sntp@2.1.0: This module moved to @hapi/sntp. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.
lerna ERR! execute npm ERR! code 1
lerna ERR! execute npm ERR! Command failed: /usr/bin/git clone -q git://github.com/debris/bignumber.js.git /home/zhuliang/.npm/_cacache/tmp/git-clone-4e247501
lerna ERR! execute npm ERR! /home/zhuliang/.npm/_cacache/tmp/git-clone-4e247501/.git: 权限不够
lerna ERR! execute npm ERR!
lerna ERR! execute
lerna ERR! execute npm ERR! A complete log of this run can be found in:
lerna ERR! execute npm ERR!     /home/zhuliang/.npm/_logs/2019-05-21T02_09_33_869Z-debug.log
lerna ERR! execute
lerna ERR! execute     at Promise.all.then.arr (/home/zhuliang/performance-caliper/caliper/node_modules/execa/index.js:210:11)
lerna ERR! execute     at <anonymous>
lerna ERR! execute     at process._tickCallback (internal/process/next_tick.js:188:7)
lerna ERR! execute  callback with error
lerna ERR! execute  { Error: Command failed: npm install
lerna ERR! execute npm WARN deprecated grpc-web-client@0.7.0: This package has been deprecated in favor of @improbable-eng/grpc-web
lerna ERR! execute npm WARN deprecated hoek@4.2.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
lerna ERR! execute npm WARN deprecated formatio@1.2.0: This package is unmaintained. Use @sinonjs/formatio instead
lerna ERR! execute npm WARN deprecated samsam@1.3.0: This package has been deprecated in favour of @sinonjs/samsam
lerna ERR! execute npm WARN deprecated text-encoding@0.6.4: no longer maintained
lerna ERR! execute npm WARN deprecated circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.
lerna ERR! execute npm WARN deprecated core-js@2.3.0: core-js@<2.6.6 is no longer maintained. Please, upgrade to core-js@3 or at least to actual version of core-js@2.
lerna ERR! execute npm WARN deprecated hawk@6.0.2: This version has been deprecated. Please upgrade to the latest version to get the best features, bug fixes, and security patches.
lerna ERR! execute npm WARN deprecated boom@4.3.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
lerna ERR! execute npm WARN deprecated cryptiles@3.1.4: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
lerna ERR! execute npm WARN deprecated boom@5.2.0: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
lerna ERR! execute npm WARN deprecated sntp@2.1.0: This module moved to @hapi/sntp. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.
lerna ERR! execute npm ERR! code 1
lerna ERR! execute npm ERR! Command failed: /usr/bin/git clone -q git://github.com/debris/bignumber.js.git /home/zhuliang/.npm/_cacache/tmp/git-clone-4e247501
lerna ERR! execute npm ERR! /home/zhuliang/.npm/_cacache/tmp/git-clone-4e247501/.git: 权限不够
lerna ERR! execute npm ERR!
lerna ERR! execute
lerna ERR! execute npm ERR! A complete log of this run can be found in:
lerna ERR! execute npm ERR!     /home/zhuliang/.npm/_logs/2019-05-21T02_09_33_869Z-debug.log
lerna ERR! execute
lerna ERR! execute     at Promise.all.then.arr (/home/zhuliang/performance-caliper/caliper/node_modules/execa/index.js:210:11)
lerna ERR! execute     at <anonymous>
lerna ERR! execute     at process._tickCallback (internal/process/next_tick.js:188:7)
lerna ERR! execute   stack: 'Error: Command failed: npm install\nnpm WARN deprecated grpc-web-client@0.7.0: This package has been deprecated in favor of @improbable-eng/grpc-web\nnpm WARN deprecated hoek@4.2.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).\nnpm WARN deprecated formatio@1.2.0: This package is unmaintained. Use @sinonjs/formatio instead\nnpm WARN deprecated samsam@1.3.0: This package has been deprecated in favour of @sinonjs/samsam\nnpm WARN deprecated text-encoding@0.6.4: no longer maintained\nnpm WARN deprecated circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.\nnpm WARN deprecated core-js@2.3.0: core-js@<2.6.6 is no longer maintained. Please, upgrade to core-js@3 or at least to actual version of core-js@2.\nnpm WARN deprecated hawk@6.0.2: This version has been deprecated. Please upgrade to the latest version to get the best features, bug fixes, and security patches.\nnpm WARN deprecated boom@4.3.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).\nnpm WARN deprecated cryptiles@3.1.4: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).\nnpm WARN deprecated boom@5.2.0: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).\nnpm WARN deprecated sntp@2.1.0: This module moved to @hapi/sntp. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.\nnpm ERR! code 1\nnpm ERR! Command failed: /usr/bin/git clone -q git://github.com/debris/bignumber.js.git /home/zhuliang/.npm/_cacache/tmp/git-clone-4e247501\nnpm ERR! /home/zhuliang/.npm/_cacache/tmp/git-clone-4e247501/.git: 权限不够\nnpm ERR! \n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /home/zhuliang/.npm/_logs/2019-05-21T02_09_33_869Z-debug.log\n\n    at Promise.all.then.arr (/home/zhuliang/performance-caliper/caliper/node_modules/execa/index.js:210:11)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:188:7)',
lerna ERR! execute   code: 1,
lerna ERR! execute   killed: false,
lerna ERR! execute   stdout: '',
lerna ERR! execute   stderr: 'npm WARN deprecated grpc-web-client@0.7.0: This package has been deprecated in favor of @improbable-eng/grpc-web\nnpm WARN deprecated hoek@4.2.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).\nnpm WARN deprecated formatio@1.2.0: This package is unmaintained. Use @sinonjs/formatio instead\nnpm WARN deprecated samsam@1.3.0: This package has been deprecated in favour of @sinonjs/samsam\nnpm WARN deprecated text-encoding@0.6.4: no longer maintained\nnpm WARN deprecated circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.\nnpm WARN deprecated core-js@2.3.0: core-js@<2.6.6 is no longer maintained. Please, upgrade to core-js@3 or at least to actual version of core-js@2.\nnpm WARN deprecated hawk@6.0.2: This version has been deprecated. Please upgrade to the latest version to get the best features, bug fixes, and security patches.\nnpm WARN deprecated boom@4.3.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).\nnpm WARN deprecated cryptiles@3.1.4: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).\nnpm WARN deprecated boom@5.2.0: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).\nnpm WARN deprecated sntp@2.1.0: This module moved to @hapi/sntp. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.\nnpm ERR! code 1\nnpm ERR! Command failed: /usr/bin/git clone -q git://github.com/debris/bignumber.js.git /home/zhuliang/.npm/_cacache/tmp/git-clone-4e247501\nnpm ERR! /home/zhuliang/.npm/_cacache/tmp/git-clone-4e247501/.git: 权限不够\nnpm ERR! \n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /home/zhuliang/.npm/_logs/2019-05-21T02_09_33_869Z-debug.log\n',
lerna ERR! execute   failed: true,
lerna ERR! execute   signal: null,
lerna ERR! execute   cmd: 'npm install',
lerna ERR! execute   timedOut: false }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! caliper@0.1.0 bootstrap: `lerna bootstrap`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the caliper@0.1.0 bootstrap script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/zhuliang/.npm/_logs/2019-05-21T02_09_34_112Z-debug.log

@aklenik sure: 
```
vagrant@localhost:/vagrant/caliper [master]$ npm i

> command-join@2.0.1 postinstall /vagrant/caliper/node_modules/command-join
> npx -p @seangenabe/tnx tnx || exit 0

npx: zainstalowano 1 w 1.011s

    Thanks for installing command-join!

    If you like this package, be sure to star its repo,
    and please consider donating:

        https://seangenabe.netlify.com/donate

npm notice created a lockfile as package-lock.json. You should commit this file.
added 297 packages from 158 contributors and audited 1357 packages in 9.36s
found 0 vulnerabilities

vagrant@localhost:/vagrant/caliper [master]$ npm run repoclean

> caliper@0.1.0 repoclean /vagrant/caliper
> lerna clean

lerna info version 2.0.0
lerna info About to remove the following directories:
lerna info packages/caliper-application/node_modules
lerna info packages/caliper-burrow/node_modules
lerna info packages/caliper-composer/node_modules
lerna info packages/caliper-core/node_modules
lerna info packages/caliper-fabric-ccp/node_modules
lerna info packages/caliper-fabric/node_modules
lerna info packages/caliper-iroha/node_modules
lerna info packages/caliper-sawtooth/node_modules
? Proceed? Yes
lerna info clean removing /vagrant/caliper/packages/caliper-application/node_modules
lerna info clean removing /vagrant/caliper/packages/caliper-burrow/node_modules
lerna info clean removing /vagrant/caliper/packages/caliper-composer/node_modules
lerna info clean removing /vagrant/caliper/packages/caliper-core/node_modules
lerna info clean removing /vagrant/caliper/packages/caliper-fabric-ccp/node_modules
lerna info clean removing /vagrant/caliper/packages/caliper-fabric/node_modules
lerna info clean removing /vagrant/caliper/packages/caliper-iroha/node_modules
lerna info clean removing /vagrant/caliper/packages/caliper-sawtooth/node_modules
lerna success clean finished
vagrant@localhost:/vagrant/caliper [master]$ npm run bootstrap

> caliper@0.1.0 bootstrap /vagrant/caliper
> lerna bootstrap

lerna info version 2.0.0
lerna info Bootstrapping 8 packages
lerna info lifecycle preinstall
lerna info Installing external dependencies
lerna info hoist Installing hoisted dependencies into root
lerna info hoist Pruning hoisted dependencies
lerna info hoist Finished pruning hoisted dependencies
lerna ERR! execute Error: Command failed: npm install
lerna ERR! execute npm WARN deprecated json3@3.3.2: Please use the native JSON object instead of JSON 3
lerna ERR! execute npm WARN deprecated formatio@1.2.0: This package is unmaintained. Use @sinonjs/formatio instead
lerna ERR! execute npm WARN deprecated samsam@1.3.0: This package has been deprecated in favour of @sinonjs/samsam
lerna ERR! execute npm WARN deprecated text-encoding@0.6.4: no longer maintained
lerna ERR! execute npm WARN deprecated hoek@4.2.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
lerna ERR! execute npm WARN deprecated core-js@2.3.0: core-js@<2.6.8 is no longer maintained. Please, upgrade to core-js@3 or at least to actual version of core-js@2.
lerna ERR! execute npm WARN deprecated grpc-web-client@0.7.0: This package has been deprecated in favor of @improbable-eng/grpc-web
lerna ERR! execute npm WARN deprecated circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.
lerna ERR! execute npm WARN deprecated hawk@6.0.2: This module moved to @hapi/hawk. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.
lerna ERR! execute npm WARN deprecated cryptiles@3.1.4: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
lerna ERR! execute npm WARN deprecated sntp@2.1.0: This module moved to @hapi/sntp. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.
lerna ERR! execute npm WARN deprecated boom@4.3.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
lerna ERR! execute npm WARN deprecated boom@5.2.0: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
lerna ERR! execute npm WARN ajv-keywords@2.1.1 requires a peer of ajv@^5.0.0 but none is installed. You must install peer dependencies yourself.
lerna ERR! execute npm WARN grpc-web-client@0.7.0 requires a peer of @types/google-protobuf@^3.2.5 but none is installed. You must install peer dependencies yourself.
lerna ERR! execute npm WARN caliper No repository field.
lerna ERR! execute npm WARN caliper No license field.
lerna ERR! execute 
lerna ERR! execute npm ERR! path /vagrant/caliper/node_modules/nyc/node_modules/yargs/node_modules/yargs-parser/package.json.3476133274
lerna ERR! execute npm ERR! code ENOENT
lerna ERR! execute npm ERR! errno -2
lerna ERR! execute npm ERR! syscall open
lerna ERR! execute npm ERR! enoent ENOENT: no such file or directory, open '/vagrant/caliper/node_modules/nyc/node_modules/yargs/node_modules/yargs-parser/package.json.3476133274'
lerna ERR! execute npm ERR! enoent This is related to npm not being able to find a file.
lerna ERR! execute npm ERR! enoent 
lerna ERR! execute 
lerna ERR! execute npm ERR! A complete log of this run can be found in:
lerna ERR! execute npm ERR!     /home/vagrant/.npm/_logs/2019-05-28T10_58_32_551Z-debug.log
lerna ERR! execute 
lerna ERR! execute     at Promise.all.then.arr (/vagrant/caliper/node_modules/execa/index.js:210:11)
lerna ERR! execute     at <anonymous>
lerna ERR! execute     at process._tickCallback (internal/process/next_tick.js:189:7)
lerna ERR! execute  callback with error
lerna ERR! execute  { Error: Command failed: npm install
lerna ERR! execute npm WARN deprecated json3@3.3.2: Please use the native JSON object instead of JSON 3
lerna ERR! execute npm WARN deprecated formatio@1.2.0: This package is unmaintained. Use @sinonjs/formatio instead
lerna ERR! execute npm WARN deprecated samsam@1.3.0: This package has been deprecated in favour of @sinonjs/samsam
lerna ERR! execute npm WARN deprecated text-encoding@0.6.4: no longer maintained
lerna ERR! execute npm WARN deprecated hoek@4.2.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
lerna ERR! execute npm WARN deprecated core-js@2.3.0: core-js@<2.6.8 is no longer maintained. Please, upgrade to core-js@3 or at least to actual version of core-js@2.
lerna ERR! execute npm WARN deprecated grpc-web-client@0.7.0: This package has been deprecated in favor of @improbable-eng/grpc-web
lerna ERR! execute npm WARN deprecated circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.
lerna ERR! execute npm WARN deprecated hawk@6.0.2: This module moved to @hapi/hawk. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.
lerna ERR! execute npm WARN deprecated cryptiles@3.1.4: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
lerna ERR! execute npm WARN deprecated sntp@2.1.0: This module moved to @hapi/sntp. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.
lerna ERR! execute npm WARN deprecated boom@4.3.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
lerna ERR! execute npm WARN deprecated boom@5.2.0: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
lerna ERR! execute npm WARN ajv-keywords@2.1.1 requires a peer of ajv@^5.0.0 but none is installed. You must install peer dependencies yourself.
lerna ERR! execute npm WARN grpc-web-client@0.7.0 requires a peer of @types/google-protobuf@^3.2.5 but none is installed. You must install peer dependencies yourself.
lerna ERR! execute npm WARN caliper No repository field.
lerna ERR! execute npm WARN caliper No license field.
lerna ERR! execute 
lerna ERR! execute npm ERR! path /vagrant/caliper/node_modules/nyc/node_modules/yargs/node_modules/yargs-parser/package.json.3476133274
lerna ERR! execute npm ERR! code ENOENT
lerna ERR! execute npm ERR! errno -2
lerna ERR! execute npm ERR! syscall open
lerna ERR! execute npm ERR! enoent ENOENT: no such file or directory, open '/vagrant/caliper/node_modules/nyc/node_modules/yargs/node_modules/yargs-parser/package.json.3476133274'
lerna ERR! execute npm ERR! enoent This is related to npm not being able to find a file.
lerna ERR! execute npm ERR! enoent 
lerna ERR! execute 
lerna ERR! execute npm ERR! A complete log of this run can be found in:
lerna ERR! execute npm ERR!     /home/vagrant/.npm/_logs/2019-05-28T10_58_32_551Z-debug.log
lerna ERR! execute 
lerna ERR! execute     at Promise.all.then.arr (/vagrant/caliper/node_modules/execa/index.js:210:11)
lerna ERR! execute     at <anonymous>
lerna ERR! execute     at process._tickCallback (internal/process/next_tick.js:189:7)
lerna ERR! execute   stack: 'Error: Command failed: npm install\nnpm WARN deprecated json3@3.3.2: Please use the native JSON object instead of JSON 3\nnpm WARN deprecated formatio@1.2.0: This package is unmaintained. Use @sinonjs/formatio instead\nnpm WARN deprecated samsam@1.3.0: This package has been deprecated in favour of @sinonjs/samsam\nnpm WARN deprecated text-encoding@0.6.4: no longer maintained\nnpm WARN deprecated hoek@4.2.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).\nnpm WARN deprecated core-js@2.3.0: core-js@<2.6.8 is no longer maintained. Please, upgrade to core-js@3 or at least to actual version of core-js@2.\nnpm WARN deprecated grpc-web-client@0.7.0: This package has been deprecated in favor of @improbable-eng/grpc-web\nnpm WARN deprecated circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.\nnpm WARN deprecated hawk@6.0.2: This module moved to @hapi/hawk. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.\nnpm WARN deprecated cryptiles@3.1.4: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).\nnpm WARN deprecated sntp@2.1.0: This module moved to @hapi/sntp. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.\nnpm WARN deprecated boom@4.3.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).\nnpm WARN deprecated boom@5.2.0: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).\nnpm WARN ajv-keywords@2.1.1 requires a peer of ajv@^5.0.0 but none is installed. You must install peer dependencies yourself.\nnpm WARN grpc-web-client@0.7.0 requires a peer of @types/google-protobuf@^3.2.5 but none is installed. You must install peer dependencies yourself.\nnpm WARN caliper No repository field.\nnpm WARN caliper No license field.\n\nnpm ERR! path /vagrant/caliper/node_modules/nyc/node_modules/yargs/node_modules/yargs-parser/package.json.3476133274\nnpm ERR! code ENOENT\nnpm ERR! errno -2\nnpm ERR! syscall open\nnpm ERR! enoent ENOENT: no such file or directory, open \'/vagrant/caliper/node_modules/nyc/node_modules/yargs/node_modules/yargs-parser/package.json.3476133274\'\nnpm ERR! enoent This is related to npm not being able to find a file.\nnpm ERR! enoent \n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /home/vagrant/.npm/_logs/2019-05-28T10_58_32_551Z-debug.log\n\n    at Promise.all.then.arr (/vagrant/caliper/node_modules/execa/index.js:210:11)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:189:7)',
lerna ERR! execute   code: 254,
lerna ERR! execute   killed: false,
lerna ERR! execute   stdout: '',
lerna ERR! execute   stderr: 'npm WARN deprecated json3@3.3.2: Please use the native JSON object instead of JSON 3\nnpm WARN deprecated formatio@1.2.0: This package is unmaintained. Use @sinonjs/formatio instead\nnpm WARN deprecated samsam@1.3.0: This package has been deprecated in favour of @sinonjs/samsam\nnpm WARN deprecated text-encoding@0.6.4: no longer maintained\nnpm WARN deprecated hoek@4.2.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).\nnpm WARN deprecated core-js@2.3.0: core-js@<2.6.8 is no longer maintained. Please, upgrade to core-js@3 or at least to actual version of core-js@2.\nnpm WARN deprecated grpc-web-client@0.7.0: This package has been deprecated in favor of @improbable-eng/grpc-web\nnpm WARN deprecated circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.\nnpm WARN deprecated hawk@6.0.2: This module moved to @hapi/hawk. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.\nnpm WARN deprecated cryptiles@3.1.4: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).\nnpm WARN deprecated sntp@2.1.0: This module moved to @hapi/sntp. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.\nnpm WARN deprecated boom@4.3.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).\nnpm WARN deprecated boom@5.2.0: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).\nnpm WARN ajv-keywords@2.1.1 requires a peer of ajv@^5.0.0 but none is installed. You must install peer dependencies yourself.\nnpm WARN grpc-web-client@0.7.0 requires a peer of @types/google-protobuf@^3.2.5 but none is installed. You must install peer dependencies yourself.\nnpm WARN caliper No repository field.\nnpm WARN caliper No license field.\n\nnpm ERR! path /vagrant/caliper/node_modules/nyc/node_modules/yargs/node_modules/yargs-parser/package.json.3476133274\nnpm ERR! code ENOENT\nnpm ERR! errno -2\nnpm ERR! syscall open\nnpm ERR! enoent ENOENT: no such file or directory, open \'/vagrant/caliper/node_modules/nyc/node_modules/yargs/node_modules/yargs-parser/package.json.3476133274\'\nnpm ERR! enoent This is related to npm not being able to find a file.\nnpm ERR! enoent \n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /home/vagrant/.npm/_logs/2019-05-28T10_58_32_551Z-debug.log\n',
lerna ERR! execute   failed: true,
lerna ERR! execute   signal: null,
lerna ERR! execute   cmd: 'npm install',
lerna ERR! execute   timedOut: false }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! caliper@0.1.0 bootstrap: `lerna bootstrap`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the caliper@0.1.0 bootstrap script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vagrant/.npm/_logs/2019-05-28T10_58_32_656Z-debug.log
```

@nklincoln This was absolute fresh Vagrant machine with Centos 7.6. 
Steps:
* Added Node8 repository:
```
[nodesource]
name=Node.js Packages for Enterprise Linux 7 - $basearch
baseurl=https://rpm.nodesource.com/pub_8.x/el/7/$basearch
failovermethod=priority
enabled=1
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/NODESOURCE-GPG-SIGNING-KEY-EL
```
* Installed Node
``` 
# yum install -y nodejs
```
* cloned Caliper repo
* npm i ; npm run repoclean ; npm run bootstrap
---------------
Just tried removing node_modules and package-lock.json as suggested. The process moved forward (progress wheel was spinning this time :)) but finished with error: 
```
vagrant@localhost:/vagrant/caliper [master]$ rm -rf node_modules 
vagrant@localhost:/vagrant/caliper [master]$ rm -rf package-lock.json 
vagrant@localhost:/vagrant/caliper [master]$ ls
CONTRIBUTING.md  ISSUE_TEMPLATE.md  LICENSE  PULL_REQUEST_TEMPLATE.md  README.md  lerna-debug.log  lerna.json  package.json  packages
vagrant@localhost:/vagrant/caliper [master]$ npm run bootstrap

> caliper@0.1.0 bootstrap /vagrant/caliper
> lerna bootstrap

lerna notice cli v3.14.1
lerna info Bootstrapping 8 packages
lerna info Installing external dependencies
lerna info hoist Installing hoisted dependencies into root
lerna info hoist Pruning hoisted dependencies
lerna info hoist Finished pruning hoisted dependencies
lerna ERR! npm install exited 254 in 'caliper'
lerna ERR! npm install stderr:
npm WARN deprecated hoek@4.2.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN deprecated json3@3.3.2: Please use the native JSON object instead of JSON 3
npm WARN deprecated formatio@1.2.0: This package is unmaintained. Use @sinonjs/formatio instead
npm WARN deprecated samsam@1.3.0: This package has been deprecated in favour of @sinonjs/samsam
npm WARN deprecated text-encoding@0.6.4: no longer maintained
npm WARN deprecated grpc-web-client@0.7.0: This package has been deprecated in favor of @improbable-eng/grpc-web
npm WARN deprecated core-js@2.3.0: core-js@<2.6.8 is no longer maintained. Please, upgrade to core-js@3 or at least to actual version of core-js@2.
npm WARN deprecated circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.
npm WARN deprecated hawk@6.0.2: This module moved to @hapi/hawk. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.
npm WARN deprecated boom@4.3.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN deprecated cryptiles@3.1.4: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN deprecated sntp@2.1.0: This module moved to @hapi/sntp. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.
npm WARN deprecated boom@5.2.0: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN ajv-keywords@2.1.1 requires a peer of ajv@^5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN grpc-web-client@0.7.0 requires a peer of @types/google-protobuf@^3.2.5 but none is installed. You must install peer dependencies yourself.

npm ERR! path /vagrant/caliper/node_modules/nyc/node_modules/yargs/node_modules/yargs-parser/package.json.3281636648
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall open
npm ERR! enoent ENOENT: no such file or directory, open '/vagrant/caliper/node_modules/nyc/node_modules/yargs/node_modules/yargs-parser/package.json.3281636648'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vagrant/.npm/_logs/2019-05-28T13_07_39_754Z-debug.log

lerna ERR! npm install exited 254 in 'caliper'
npm ERR! code ELIFECYCLE
npm ERR! errno 254
npm ERR! caliper@0.1.0 bootstrap: `lerna bootstrap`
npm ERR! Exit status 254
npm ERR! 
npm ERR! Failed at the caliper@0.1.0 bootstrap script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vagrant/.npm/_logs/2019-05-28T13_07_39_864Z-debug.log
```
And debug.log
```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'run', 'bootstrap' ]
2 info using npm@6.4.1
3 info using node@v8.16.0
4 verbose run-script [ 'prebootstrap', 'bootstrap', 'postbootstrap' ]
5 info lifecycle caliper@0.1.0~prebootstrap: caliper@0.1.0
6 info lifecycle caliper@0.1.0~bootstrap: caliper@0.1.0
7 verbose lifecycle caliper@0.1.0~bootstrap: unsafe-perm in lifecycle true
8 verbose lifecycle caliper@0.1.0~bootstrap: PATH: /usr/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/vagrant/caliper/node_modules/.bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/home/vagrant/.local/bin:/home/vagrant/bin
9 verbose lifecycle caliper@0.1.0~bootstrap: CWD: /vagrant/caliper
10 silly lifecycle caliper@0.1.0~bootstrap: Args: [ '-c', 'lerna bootstrap' ]
11 silly lifecycle caliper@0.1.0~bootstrap: Returned: code: 254  signal: null
12 info lifecycle caliper@0.1.0~bootstrap: Failed to exec bootstrap script
13 verbose stack Error: caliper@0.1.0 bootstrap: `lerna bootstrap`
13 verbose stack Exit status 254
13 verbose stack     at EventEmitter.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:301:16)
13 verbose stack     at emitTwo (events.js:126:13)
13 verbose stack     at EventEmitter.emit (events.js:214:7)
13 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at emitTwo (events.js:126:13)
13 verbose stack     at ChildProcess.emit (events.js:214:7)
13 verbose stack     at maybeClose (internal/child_process.js:915:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
14 verbose pkgid caliper@0.1.0
15 verbose cwd /vagrant/caliper
16 verbose Linux 3.10.0-957.1.3.el7.x86_64
17 verbose argv "/usr/bin/node" "/usr/bin/npm" "run" "bootstrap"
18 verbose node v8.16.0
19 verbose npm  v6.4.1
20 error code ELIFECYCLE
21 error errno 254
22 error caliper@0.1.0 bootstrap: `lerna bootstrap`
22 error Exit status 254
23 error Failed at the caliper@0.1.0 bootstrap script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 254, true ]
```

--- a/docs/1_Getting_Started.md
+++ b/docs/1_Getting_Started.md
@@ -32,7 +32,7 @@ See [Architecture Introduction]({{ site.baseurl }}{% link docs/2_Architecture.md
 ## Pre-requisites
 
 Make sure following tools are installed
-* NodeJS 8.X
+* NodeJS 8 (LTS), 9, or 10 (LTS) *we do not support higher versions as the dependancy chain does not permit this*
 * node-gyp
 * Docker
 * Docker-compose
@@ -44,6 +44,8 @@ Caliper is split into pacakges that are managed by Lerna, a tool for managing Ja
 - Run `npm install` in Caliper root folder to install base dependencies locally
 - Run `npm run repoclean` in Caliper root folder to ensure that all the packages are clean
 - Run `npm run bootstrap` to bootstrap the packages in the Caliper repository. This will install all package dependancies and link any cross dependancies. It will take some time to finish installation. If it is interrupted by `ctrl+c`, please recover the file `package.json` first and then run `npm run bootstrap` again.
+
+Do not run any of the above commands with `sudo`, as it will cause the bootstrap process to fail
 
 Steps for configuring a benchmark that targets a supported blockchain technology are given in the following links:
 

